lib/utils/lifecycle.js

```diff
@@ -3,14 +3,16 @@ exports.cmd = cmd
 exports.makeEnv = makeEnv
 
 var log = require("npmlog")
-  , spawn = require("./spawn")
-  , npm = require("../npm.js")
-  , path = require("path")
-  , fs = require("graceful-fs")
-  , chain = require("slide").chain
-  , Stream = require("stream").Stream
-  , PATH = "PATH"
-  , uidNumber = require("uid-number")
+var spawn = require("./spawn")
```

Comment:
I'm probably going to semi-revert this hunk -- style changes will be done atomically, per @isaacs's recommendations, and until then, I don't want to noisy up the diff.